src/css/ds-vars.css

```diff
@@ -1,6 +1,6 @@
 :root {
   --ds-common-black: #090909;
-  --ds-common-white: #fff;
+  --ds-common-white: #ff0;
```

Comment:
I'll remove this before merging, but needed a way to test.

Reply:
My eyes 😵

Reply:
:joy: